docs/content/preview/explore/query-1-performance/pg-stat-progress-copy.md

```diff
@@ -0,0 +1,177 @@
+
+In PostgreSQL, after the `COPY` command is finished then the row containing details of the copy command is _removed_ from the view. In YugabyteDB, information is _retained_ in the view after the copy has finished. Also, information is retained only for the last executed copy command on that connection.
+
+This is required for YugabyteDB, because if the `COPY` command finishes due to an error, then you would like to know how many rows were reliably persisted to the disk.
+
+## Examples
+
+The following examples demonstrates the possible stages (IN PROGRESS, ERROR, SUCCESS) for the copy operation.
+
+### Prerequisites
+
+- Make sure you have YugabyteDB up and running. If you are new to YugabyteDB, follow the steps in [Quick start](../../../quick-start/) to have YugabyteDB up and running in minutes.
+
+- Run the following script from your YugabyteDB installation directory to generate data to a file `test_data.csv`.
+
+    ```sh
+    #!/bin/bash
+    rm -rf test_data.csv
+    for i in {1..100000}
+    do
+       echo "$i, $i, $i" >> test_data.csv
+    done
+    ```
+
+- From your local YugabyteDB installation directory, connect to the [YSQL](../../../../admin/ysqlsh/) shell, and create a table using the following schema:
+
+    ```sql
+    create table test_copy ( h1 int, r1 int, v1 int, primary key (h1, r1));
+    ```
+
+### Run a successful COPY command
+
+1. Copy `test_data.csv` using the following copy command:
+
+    ```sql
+    \copy test_copy from 'test_data.csv'  (DELIMITER ',');
+    ```
+
+    ```output
+    COPY 100000
+    ```
+
+1. Verify `yb_status` using the view.
+
+    ```sql
+    select * from pg_stat_progress_copy ;
+    ```
+
+    You should see output similar to the following:
+
+    ```output
+     pid  | datid | datname  | relid |  command  | type | yb_status | bytes_processed | bytes_total | tuples_processed | tuples_excluded
+    -------+-------+----------+-------+-----------+------+-----------+-----------------+-------------+------------------+-----------------
+    74390 | 13288 | yugabyte | 16394 | COPY FROM | PIPE | SUCCESS   |         1966685 |           0 |           100000 |               0
+    (1 row)
+    ```
+
+### Verify a COPY command status in progress
+
+Copy `test_data.csv` using the following copy command and check the status in parallel from another terminal.
+
+```sql
+\copy test_copy from 'test_data.csv'  (DELIMITER ',');
+```
+
+```sql
+select * from pg_stat_progress_copy ;
+```
+
+You should see output similar to the following:
+
+```output
+ pid  | datid | datname  | relid |  command  | type |  yb_status  | bytes_processed | bytes_total | tuples_processed | tuples_excluded
+-------+-------+----------+-------+-----------+------+-------------+-----------------+-------------+------------------+-----------------
+74390 | 13288 | yugabyte | 16404 | COPY FROM | PIPE | IN PROGRESS |          766682 |           0 |            40000 |               0
+(1 row)
+```
+
+### Verify an interrupted COPY operation
```

Comment:
Lets add that the  default for ROWS_PER_TRANSACTION is 20k, and interruption happened ROW 55184 in the given example, so last 15184 rows were not persisted and first 40K rows were persisted.